source/administration/image-proxy.rst

```diff
@@ -0,0 +1,59 @@
+Image Proxy
+================================
+
+Using image proxies means that users make only secure requests. By allowing image requests to go straight to third-party
+servers, tracking pixels is allowed. Users can put invisible images in posts that logs time and location data
+for every user that views the post. An image proxy protects user privacy by eliminating this direct interaction with 
+third-party servers.
+
+Many proxy servers also provide resizing-as-a-service. For example, if you use `willnorris/imageproxy`, you can configure 
```

Comment:
I had to Google who/what "willnorris" is...  Would it clarify to end-users if we included the full GitHub address e.g. "https://github.com/willnorris/imageproxy" so they realise it's image proxy software written by the person "Will Norris"?

Reply:
Didn't we remove support for willnorris in 4.7? Maybe remove it from the docs as an example if so? (JB)